README.md

```diff
@@ -106,44 +110,46 @@ Other shortcuts:
 | Alt+Enter | Fullscreen (DirectX) |
 | Ctrl+R | Reset |
 
-Currently, DirectX 11 and OpenGL are supported. Change the renderer by opening the `shipofharkinian.json` configuration file in notepad and add `sdl` to the quotes in `"GfxBackend": ""` for OpenGL or leave blank for DirectX.
+Currently, DirectX 11 and OpenGL are supported. You can change which API to use in the `Settings` menu of the menubar, which requires a restart.  If you're having an issue with crashing, you can change the API in the `shipofharkinian.json` file by finding the line `gfxbackend:""` and changing the value to `sdl` for OpenGL. DirectX 11 is the default on Windows.
 
 ### Custom Music
 
-We support importing custom [Seq64](https://github.com/sauraen/seq64) files to replace the in game music and fanfares (Not sound effects yet currently).
+We support importing custom [Seq64](https://github.com/sauraen/seq64) files to replace the in game music and fanfares (SFX replacement is currently not supported).
```

Comment:
I'd recommend spelling SFX out completely `(replacing sound effects with custom ones is currently not supported)`

Reply:
Yeah, considering the SFX editor is named as such, it could be confusing. Thank you!